spec/models/submission_entry_spec.rb

```diff
@@ -31,4 +31,14 @@
       expect(SubmissionEntry.wider_public_sector).to contain_exactly(bobs_charity_entry)
     end
   end
+
+  describe 'ordered_by_row' do
+    let!(:third_row)  { FactoryBot.create(:submission_entry, source: { 'row' => 3 }) }
+    let!(:first_row)  { FactoryBot.create(:submission_entry, source: { 'row' => 1 }) }
+    let!(:second_row) { FactoryBot.create(:submission_entry, source: { 'row' => 2 }) }
```

Comment:
I don't believe the execution order matters here, mainly because we're using UUIDs as the primary key. therefor the default ordering will always return the entries in a pretty much random order, regardless of the order the records are created in. what do you think?